client/state/stats/lists/test/selectors.js

```diff
@@ -183,4 +196,43 @@ describe( 'selectors', () => {
 			expect( stats ).to.eql( 2 );
 		} );
 	} );
+
+	describe( 'getSiteStatsPostsCountByDay()', () => {
```

Comment:
Speaking of memoizing, if you're going to keep it, we'll want to reset before each test.